rules/any-pass-simplification.js

```diff
@@ -1,24 +1,24 @@
 'use strict';
+const R = require('ramda');
+const isCalling = require('../ast-helper').isCalling;
```

Comment:
This file needs to be added to the whitelist of published files in `package.json`.

Reply:
Fixed.